srcpkgs/zelbar/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'zelbar'
+pkgname=zelbar
+version=1.1.0
+revision=1
+build_style=zig-build
+hostmakedepends="git pkgconf"
```

Comment:
git shouldn't be needed, the version is hardcoded in the build system, also use pkg-config instead of pkgconf